src/ui/events.js

```diff
@@ -73,22 +73,36 @@ export class MapMouseEvent extends Event {
      */
     lngLat: LngLat;
 
-    /**
+    /**.
      * If a `layerId` was specified when adding the event listener with {@link Map#on}, `features` will be an array of
      * [GeoJSON](http://geojson.org/) [Feature objects](https://tools.ietf.org/html/rfc7946#section-3.2).
      * The array will contain all features from that layer that are rendered at the event's point.
      * The `features` are identical to those returned by {@link Map#queryRenderedFeatures}.
      *
+     *
+     * If [`layerId1`, `layerId2`] were specified when adding the event listener with {@link Map#on}, `features` will be an array of
+     * [GeoJSON](http://geojson.org/) [Feature objects](https://tools.ietf.org/html/rfc7946#section-3.2).
+     * The array will contain all features from these layers that are rendered at the event's point.
+     * The `features` are identical to those returned by {@link Map#queryRenderedFeatures}.
```

Comment:
Instead of copy-pasting the same section of text I'd collapse everything  you have between LN 77 - 86 into one paragraph.
```
If a single `layerId`(as a single string ) or multiple `layerIds` (as an array of strings) was specified with {@link Map#on}, `features` will be an array of [GeoJSON](http://geojson.org/) [Feature objects](https://tools.ietf.org/html/rfc7946#section-3.2). The array will contain all features from that layer that are rendered at the event's point. The `features` are identical to those returned by {@link Map#queryRenderedFeatures}. The `features` array is ordered based on the order in which the features are rendered, with the topmost feature being at the top of the array.
```